Frontend/web/src/styles/globals.css

```
@@ -161,6 +197,624 @@ body {
+    padding: 6px;
+    box-shadow: 0 24px 56px -16px rgba(15, 23, 42, 0.25);
+  }
+
+  /* Entry stagger */
+  .hero-fade-up {
+    opacity: 0;
+    transform: translateY(8px);
+    animation: heroFadeUp 520ms cubic-bezier(0.22, 1, 0.36, 1) forwards;
+  }
+
+  @keyframes heroFadeUp {
```

Stylelint (17.12.0)

[error] Expected keyframe name "heroFadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)